test/parallel/test-v8-serdes.js

```diff
@@ -118,3 +118,11 @@ const objects = [
   assert.deepStrictEqual(buf, ser.releaseBuffer());
   assert.strictEqual(des.getWireFormatVersion(), 0x0d);
 }
+
+{
+  // Unaligned Uint16Array read, with padding in the underlying array buffer.
+  const buf = Buffer.from('0'.repeat(64) + 'ff0d5c0404addeefbe', 'hex')
+      .slice(32);
```

Comment:
@bnoordhuis Yes, that’s the idea. I’ve updated the test with a variant of your suggestion